x-pack/plugins/cases/server/services/transform.ts

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
🔴 : Shared transformation logic between the case and configuration saved objects